app/api/routers/token.py

```diff
@@ -549,3 +557,164 @@ def list_all_transfer_approval_histories(
         "data": data
     }
 
+
+@router.get(
+    "/{token_address}/Lock",
+    summary="Token Lock Events",
+    operation_id="Lock",
+    response_model=GenericSuccessResponse[LockEventsResponse],
+    responses=get_routers_responses(DataNotExistsError, InvalidParameterError)
+)
+def list_all_lock_events(
```

Comment:
Add event list endpoints for lock and unlock.

Reply:
GET: `/Token/{token_address}/Lock`/ `/Token/{token_address}/Unlock` are removed:  https://github.com/BoostryJP/ibet-Wallet-API/pull/1293/commits/5cc1a101083132bbba5632db112c61b48ac79443